dandi/cli/cmd_organize.py

```diff
@@ -34,6 +36,20 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--rewrite",
+    type=click.Choice(["external-file"]),
+    default=None,
+    help="Use to re-write attributes in an nwbfile. For example, if the value "
+    "is set to 'external-paths' then the external_path attribute of nwbfiles' "
+    "ImageSeries will be written with renamed video paths according to set rules.",
+)
+@click.option(
+    "--external-files-mode",
+    type=click.Choice(file_operation_modes),
+    default=None,
+    help="check help for --files-mode option",
```

Comment:
Rewrite this help string to something like "Like `--files-mode` but [explain difference here]".

Reply:
I have made changes here https://github.com/dandi/dandi-cli/pull/841/commits/d37a9758a8600c2bcd3f0370850a99133c0a5c32

Reply:
That's still not good enough.  The help strings should give the user an idea when they would want to use `--files-mode` versus `--external-files-mode`.  What does `--external-files-mode` do that `--files-mode` doesn't?

Secondly, if `--external-file-mode` can only be "move" or "copy", then its `click.Choice` values should be limited to those instead of being `file_operation_modes`.